enaml/core/parser/base_parser.py

```diff
@@ -497,29 +496,23 @@ def p_pragma_arg3(self, p):
     # -------------------------------------------------------------------------
     # AliasExpr
     # -------------------------------------------------------------------------
-    def p_alias_expr1(self, p):
+    def d_alias_expr1(self, p):
         ''' alias_expr : ALIAS NAME NEWLINE '''
         node = enaml_ast.AliasExpr()
         node.lineno = p.lineno(1)
         node.name = p[2]
         node.target = p[2]
         p[0] = node
 
-    def p_alias_expr2(self, p):
-        ''' alias_expr : ALIAS NAME COLON NAME NEWLINE '''
+    def d_alias_expr2(self, p):
+        ''' alias_expr : ALIAS NAME COLON dotted_name NEWLINE '''
```

Comment:
why are you removing support for `NAME`? You can alias something that isn't a dotted name.